fv3/fv3_cap.F90

```diff
@@ -952,7 +952,7 @@ subroutine InitializeAdvertise(gcomp, rc)
               if( output_startfh == 0) then
                 output_fh(1) = dt_atmos/3600.
               else
-                output_fh(1) = output_startfh
+                output_fh(1) = output_startfh + dt_atmos/3600.
```

Comment:
No need for this if test, if `output_startfh` is zero `output_fh(1) = output_startfh + dt_atmos/3600.` will be equal to `output_fh(1) = dt_atmos/3600.`

Reply:
updated.